### PR TITLE
Add server kafka usage

### PR DESCRIPTION
**Pull request by lamr02n**
The `LogServer` can now simultaneously receive on a Kafka topic and on its open receiving port.

### CODE REVIEW

File: src/logserver/server.py [hunk 1]
```diff
@@ -133,6 +147,8 @@ def get_next_logline(self) -> str | None:
             return self.data_queue.get()
         return None
 
+    # TODO: Add a close method
```
Comment (stefanDeveloper) [inline src/logserver/server.py:150]: Is this still open?

**Review by stefanDeveloper — APPROVED** [review]
Looks good

File: src/logserver/server.py (hunk 1 above)
Comment (lamr02n) [reply src/logserver/server.py:150]: It's not crucial, but yes, it's still open.

Comment (stefanDeveloper) [reply src/logserver/server.py:150]: We could handle this in a later pull request if desired

Comment (lamr02n) [reply src/logserver/server.py:150]: Yes, we should. Closing the `LogServer` might still lead to some unexpected behavior. #24

**Review by stefanDeveloper — APPROVED** [review]

**Pull request merged**